base/process.jl

```diff
@@ -72,6 +72,13 @@ backticks, e.g.
 """
 Cmd
 
+hash(x::Cmd) = hash(x.exec, hash(x.env, hash(x.ignorestatus, hash(x.dir, hash(x.flags)))))
+function ==(x::Cmd, y::Cmd)
+    a = isequal(x.exec, y.exec) && isequal(x.env, y.env)
```

Comment:
was trying to follow https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#general-formatting-guidelines-for-julia-code-contributions - when I put them all on the same line it was past the 92 char line length limit so I spread it over several lines